java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/SymbolicValue.java

```diff
@@ -161,6 +162,12 @@ public SymbolicValue wrappedValue() {
   public abstract static class UnarySymbolicValue extends SymbolicValue {
     protected SymbolicValue operand;
 
+    public UnarySymbolicValue() {
+    }
+
+    public UnarySymbolicValue(SymbolicValue operand) {
```

Comment:
side note : no need for modification but I tend to think we could have skipped the introduction of constructors.